ice_setup/ice.py

```diff
@@ -567,6 +575,7 @@ def create_repo_file(cls, template_name, repo_url, gpg_url, file_name=None, **kw
             contents = template.format(
                 gpg_url=gpg_url,
                 repo_url=repo_url,
+                gpg_check=1 if use_gpg else 0,
```

Comment:
I don't feel strongly about this, but *my* rule of thumb is:

If the signature already allows `**kw` and the new keyword argument is:
* a one off, or one that is not used all the time

and/or:

* adding more arguments to the function is making a the signature too long

For documenting purposes, well, that is what docstrings are for :)

Again, just a nitpick, this should not halt this from moving forward.